node/src/components/linear_chain_sync/peers.rs

```diff
@@ -0,0 +1,54 @@
+use datasize::DataSize;
+use rand::{seq::SliceRandom, Rng};
+
+#[derive(DataSize, Debug)]
+pub struct PeersState<I> {
+    // Set of peers that we can requests block from.
```

Comment:
```suggestion
    /// Set of peers that we can request blocks from.
```

Reply:
Done in [`77e0123` (#783)](https://github.com/CasperLabs/casper-node/pull/783/commits/77e01237332b715fb645ee57771ed17afff9c7e9)